### PR TITLE
Make warnings on translate page more consistent in tests

### PR DESCRIPTION
**Pull request by mythmon**
This makes the tests pass consistently for me, and I think the logic is unchanged (though the result is a bit surprising).

@safwanrahman r?

@homu delegate=safwanrahman

### CODE REVIEW

**Comment by mythmon** [comment]
@homu priority=1

**Comment by safwanrahman** [comment]
I think, Changing the way it test is a better approach.
Rather than counting the number, we should check if there is the warning message. 
So add a class [here](https://github.com/mozilla/kitsune/blob/master/kitsune/wiki/jinja2/wiki/translate.html#L59) and in test, check if the class is present. That makes more sense.

**Comment by mythmon** [comment]
I made those changes ^

File: kitsune/wiki/views.py [hunk 1]
```diff
@@ -1517,8 +1517,7 @@ def _get_next_url_fallback_localization(request):
 
 def _show_revision_warning(document, revision):
     if revision:
-        return document.revisions.filter(created__gt=revision.created,
-                                         reviewed=None).exists()
+        return document.revisions.filter(id__gt=revision.id, reviewed=None).exists()
```
Comment (safwanrahman) [inline kitsune/wiki/views.py]: This one is Awesome for Future work. Thanks a lot!

**Comment by safwanrahman** [comment]
@homu r+

**Comment by mythmon** [comment]
Homu seems to have forgotten about us.

**Pull request merged**